docs/usecases.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Use cases
+weight: 10
+description: >
+  Use cases for the Seccomp Agent.
+---
+
+There are several possible use cases for the Seccomp Agent. Not all of them are
+implemented.
+
+## Mounting procfs in unprivileged containers
+
+- unprivileged container builds (procfs mounts with masked entries)
+
+## Support for a subset of device mknod
+
+A VPN container might need `/dev/net/tun` but cannot create the device without
+`CAP_MKNOD`. Giving this capability to the container could be risky: the
+container would be able to abuse the mknod call to get access to disks such as
+`/dev/sda`.
+
+The alternative could be to keep the container without `CAP_MKNOD` and add a
+seccomp filter on `mknod` to let the Seccomp Agent run `mknod()` on behalf of
+the container,
+
+## Userspace emulation of idmapped mounts
```

Comment:
The goal of subuidless is to allow running containers without /etc/subuid, which isn't good fit for shared environments. Almost unrelated to idmapped mounts.

Reply:
Thanks! I updated it.